test/uk/gov/hmrc/sdes/bulkdownload/acceptance/BulkDownloadListSpec.scala

```diff
@@ -78,12 +78,13 @@ class BulkDownloadListSpec extends PlaySpec with GuiceOneAppPerSuite with WireMo
 
 
     "no files are found i.e. SDES proxy returns an empty array" should {
-      "respond with 404 NotFound" in {
+      "respond with 200 OK with empty content" in {
```

Comment:
Optional: `empty list` sounds more feasible that `empty content`

Reply:
incorporated all the comments